__tests__/fixtures/install/install-production-bin/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "devDependencies": {
```

Comment:
This is an excessively large node_modules that gets installed when the test is running.
Could you find lighter ones for the test?